library/CMakeLists.txt

```diff
@@ -20,7 +20,7 @@
 # ########################################################################
 
 # This is incremented when the ABI to the library changes
-set( hipblas_SOVERSION 2.0 )
+set( hipblas_SOVERSION 1.0 )
```

Comment:
Same question, wouldn't 1.1 be better.

Reply:
Thanks, changed hipblas_SOVERSION to 1.1.0